solvers/condat_linear.py

```diff
@@ -0,0 +1,39 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+from benchopt.stopping_criterion import SingleRunCriterion
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from benchmark_utils.tv_numba import linearized_taut_string
+
+
+class Solver(BaseSolver):
+    """Solve the TV minimization problem using a majorization-minimization algorithm.
+
+    Condat, L. (2013). A direct algorithm for 1D total variation denoising.
+    https://github.com/albarji/proxTV/blob/master/src/TVL1opt.cpp
+    """
+
+    name = "Linearized Taut String"
+    stopping_criterion = SingleRunCriterion()
```

Comment:
do you need this?

Reply:
Yep, this method is not iterative (only one pass on the data).